accounts/static/accounts/css/onboarding_movies.css

```diff
@@ -106,40 +101,42 @@
   transition: all 0.3s ease;
 }
 
-@media only screen and (max-width: 480px) {
-  .tooltip_button:hover {
+@media only screen and (width <= 480px) {
+  .tooltip-button:hover {
     background-color: #9a8aff;
     transform: scale(1) !important;
   }
-
 }
 
-@media screen and (max-width: 500px) {
-  #help_modal{
-    width:100% !important; /*TODO: overrides separate !important width in modal.css  */
-    padding: 20px 0px;
+@media screen and (width <= 500px) {
+  #help-modal {
+    width: 100% !important; /* TODO: overrides separate !important width in modal.css  */
+    padding: 20px 0;
```

Comment:
"In CSS, 0 is treated the same as 0px, so both rules are functionally identical."

didn't know that, thanks for telling me that gpt.